source/jormungandr/tests/graphical_isochrones_tests.py

```diff
@@ -219,6 +219,25 @@ def test_graphical_isochrone_multi_isochrone(self):
 
         assert len(response['isochrones']) == 7
 
+    def test_graphical_isochrones_forbidden_uris(self):
+        basic_query = "v1/coverage/main_routing_test/isochrones?from={}&datetime={}&max_duration={}"
+        basic_query = basic_query.format(s_coord, "20120614T080000", "300")
+        basic_response = self.query(basic_query)
+        q_forbidden_uris = basic_query + "&forbidden_uris[]=A"
+        q_forbidden_uris = q_forbidden_uris.format(s_coord, "20120614T080000", "300")
+        response_forbidden_uris = self.query(q_forbidden_uris)
+
+        is_valid_graphical_isochrone(basic_response, self.tester, basic_query)
+        is_valid_graphical_isochrone(response_forbidden_uris, self.tester, basic_query)
+
+        for basic_isochrone, isochrone_forbidden_uris in zip(basic_response['isochrones'],
+                                                             response_forbidden_uris['isochrones']):
+            basic_multi_poly = asShape(basic_isochrone['geojson'])
+            multi_poly_forbidden_uris = asShape(isochrone_forbidden_uris['geojson'])
+
+        assert not basic_multi_poly.contains(multi_poly_forbidden_uris)
```

Comment:
Why not? with only a max_duration and less vehicle, your shape must be included, no?